oval_graph/command_line.py

```diff
@@ -9,18 +9,18 @@ def print_where_is_saved_result(results_src):
 
 
 def main():
-    client = oval_graph.client.client(sys.argv[1:])
-    rules = client.search_rules_id()
+    client_ = client(sys.argv[1:])
```

Comment:
I don't like the variable name `client_`. What about making the current class name `client` uppercase `Client` according to the standard? The, you could have the variable `client` as before.